ga4gh/frontend.py

```diff
@@ -41,49 +43,58 @@ def handleHTTPOptions():
     return response
 
 
+@app.errorhandler(Exception)
+def handle_error(error):
+    if not isinstance(error, exceptions.ParentException):
+        error = exceptions.ServerException()
+    response = flask.jsonify(error.toDict())
```

Comment:
We should use the `protocol.GAException` class here, and remove the `toDict()` method. Something like
```python
protocolException = protocol.GAException()
# Or should our exception class use code and message for consistencty?
protocolException.errorCode = error.code
protocolException.message = error.description
response = flask.jsonify(error.toJSONDict())
```
We also need to make sure we're not losing the stack trace when unexpected errors occur during development. Maybe a development/debug flag would be useful for this?